units/src/amount/tests.rs

```diff
@@ -879,50 +900,6 @@ fn serde_as_str_opt() {
     assert_eq!(without, serde_json::from_value(value_without).unwrap());
 }
 
-#[test]
-fn sum_amounts() {
```

Comment:
Maybe this will be apparent as I look at this more, but why is this being removed?